scripts/converter.js

```diff
@@ -0,0 +1,5 @@
+class Converter {
+  static convertToArray(string, seperator) {
```

Comment:
In order to make this function easier to use I'd opt for a default argument. :) Also you spelled separator wrong. :*

```javascript
static convertToArray(string, separator = ';') {
```

Reply:
I did indeed, and I will fix that :))